packages/charts/src/vaadin-chart-series.js

```diff
@@ -8,9 +8,10 @@
  * See https://vaadin.com/commercial-license-and-service-terms for the full
  * license.
  */
+import './vaadin-chart.js';
```

Comment:
I think there is no need for this import. In `vaadin-chart.js` we import `vaadin-chart-series.js` and that should be enough. Previously the import was there only to get global `Chart` class for `instanceof` check.

Reply:
Done